ast/misc.go

```diff
@@ -318,10 +318,12 @@ func (n *FlushStmt) Accept(v Visitor) (Node, bool) {
 type KillStmt struct {
 	stmtNode
 
+	// Query ...
 	// If Query is true, terminates the statement the connection is currently executing, but leaves the connection itself intact.
 	// If Query is false, terminates the connection associated with the given ConnectionID, after terminating any statement the connection is executing.
 	Query        bool
 	ConnectionID uint64
+	// TiDBExtension ...
```

Comment:
TiDBExtension is used to indicate whether the user knows he is sending kill statement to the right tidb-server.